app/views/responses/results.html.erb

```diff
@@ -19,7 +19,7 @@
     <div class="flex-2-thirds">
       <div class="block--results">
         <h2><%= t('results.title1') %></h2>
-        <div><%= t('results.content1') %></div>
+        <div><%= t('results.content1').html_safe %></div>
```

Comment:
@WKocur I am just wondering about this variable name `content1` and whether it would be better to name it somehow without the number at the end? Or I am unsure of the convention myself! Please let me know otherwise.